packages/marko-web/components/page/metadata/google-structured-data/content.js

```diff
@@ -49,6 +55,7 @@ module.exports = (node) => {
       uploadDate: publishedISOString,
       contentUrl: get(node, 'siteContext.canonicalUrl'),
       embedUrl: get(node, 'embedSrc'),
+      ...(get(node, 'transcript') && { transcript: convertHtmlToText(get(node, 'transcript')) }),
```

Comment:
What happens if the user puts in a bad or empty value? If this throws, does the entire page error or does the JSON+LD data break quietly?

Reply:
@solocommand  I added a value check to the converthHtmlToSring function & I also added a check to make sure there is a returned value.  In testing or trying to make it break I was only able to make cheerio break by not sending a value at all.  It converted broken or crappy html just fine.  I can add some additional check but not sure if you want me to wrap the cheerio.load() in some sort of try catch?  Thoughts?